src/Tasks/Task-3 (Redux)/actions/index.js

```diff
@@ -0,0 +1,13 @@
+export const incNumber = (num) => {
+    return {
+        type:"INCREMENT",
```

Comment:
Manage variables for the attribute like INCREMENT and DECREMENT in another file and import here and use.